decidim-accountability/app/commands/decidim/accountability/admin/create_result.rb

```diff
@@ -43,7 +43,8 @@ def create_result
             start_date: @form.start_date,
             end_date: @form.end_date,
             progress: @form.progress,
-            decidim_accountability_status_id: @form.decidim_accountability_status_id
+            decidim_accountability_status_id: @form.decidim_accountability_status_id,
+            external_id: @form.external_id
```

Comment:
Missing form input?

Reply:
Do you mean that the form isn't updated? It is here: https://github.com/decidim/decidim/pull/2493/files#diff-cd7ed135a4238adb5af1768b7f26c353

Reply:
No, I mean the view is not being updated, so the form input will not be rendered, the users will never be able to fill it and this field will always be blank

Reply:
Right! I forgot the UI because we are importing the results using the importer. I'll fix it ASAP!